markdown/guide/addons.md

```diff
@@ -23,6 +23,16 @@ ember-engines will automatically try to deduplicate addons used by your host app
 
 If addons are not included in the host app, they will be included in the engine's vendor bundle. 
 
+Isolation and liability is generally hard to solve, all possible cases have down sides here:
+
+ 1) Include shared deps into the host app - Downside: Increases initial bundle size.
+
+ 2) Include shared deps into each engine bundle - Downside: Duplicated stuff in multiple engine bundles.
+
+ 3) Somehow split out shared deps into dedicated bundles `(e.g. /engine-assets/shared-by-a-and-b.js)` - Downside: You might end up with 20 bundles `(e.g. shared-by-a-and-b, shared-by-b-and-c-, shared-by-a-b-and-c, ...)`, and it is probably pretty complicated from a built step perspective
+
+The **case 1** is currently the best approach
+
```

Comment:
IMO, the above section doesn't really work following the prior two paragraphs, which explain what ember-engines actually does. It would make more sense as a lead-in to the problem that starts the section.

Also, saying "The **case 1** is currently the best approach" isn't particularly useful. This could be dropped in favor of describing the approach that ember-engines takes and why.

Reply:
> IMO, a seção acima não funciona realmente seguindo os dois parágrafos anteriores, que explicam o que os motores de ember realmente fazem. Isso faria mais sentido como uma introdução ao problema que inicia a seção.

I'll improve it

sorry the **CASE 2** is the best approach today. We talked about this in some issues in the ember engines repo

Reply:
fixed @dgeb